File: data-structures-algorithms-coursera/course1-algorithmic-toolbox/week1_programming_challenges/1_sum_of_two_digits/APlusB.cpp
// Input format: Integers a and b on the same line (separated by a space).
// Output format: The sum of a and b.

#include <iostream>

int sum_of_two_digits(int x, int y) {
    return x + y;
}

int main() {
    int x = 0;
    int y = 0;
    std::cin >> x;
    std::cin >> y;
    std::cout << sum_of_two_digits(x, y);
    return 0;
}

File: data-structures-algorithms-coursera/course1-algorithmic-toolbox/week1_programming_challenges/1_sum_of_two_digits/APlusB.py
# Input format: Integers a and b on the same line (separated by a space).
# Output format: The sum of a and b.

def sum_of_two_digits(first_digit, second_digit):
    return first_digit + second_digit

def to_int(a):
    return int(a)

if __name__ == '__main__':
    a, b = map(to_int, input().split())
    print(sum_of_two_digits(a, b))
